content/docs/reactivesearch/v3/map/reactivegooglemap.md

```diff
@@ -114,54 +114,61 @@ Example uses:
     CSS class to be injected on the component container.
 -   **style** `Object`
     CSS style object to be applied to the `ReactiveGoogleMap` component.
--   **renderData** `function`
+-   **renderItem** `function`
     event fired when one or more markers are indexed, updated or removed from the map. It takes an object with the following formats (`label`, `icon`, `custom`):
 
 ```js
 // To render the given text in the marker
-renderData={result => ({
+renderItem={result => ({
     label: result.title,
 })}
 
 // To render a marker image
-renderData={result => ({
+renderItem={result => ({
     icon: 'https://i.imgur.com/NHR2tYL.png',
 })}
 
 // To render a custom markup (as label) in the marker position
-renderData={result => ({
+renderItem={result => ({
     custom: (<div>{result.mag}</div>),
 })}
 ```
 
--   **renderAllData** `function`
+-   **render** `function`
     use to display results and map component together. Usage:
```

Comment:
@mohdashraf010897 Add details for all the properties. You can follow the `ReactiveList` docs.